deps/chakrashim/chakracore.gyp

```diff
@@ -110,6 +110,7 @@
                 '/p:Configuration=$(ConfigurationName)',
                 '/p:RuntimeLib=<(component)',
                 '/p:AdditionalPreprocessorDefinitions=COMPILE_DISABLE_Simdjs=1',
+                '/p:WindowsTargetPlatformVersion=$(WindowsTargetPlatformVersion)',
```

Comment:
I think this is needed in general for VS2017. Just happened to read [this](https://github.com/Microsoft/ChakraCore/issues/2348#issuecomment-304963588)